src/main/scala/org/hammerlab/guacamole/loci/partitioning/LociPartitioner.scala

```diff
@@ -18,10 +18,12 @@ trait LociPartitionerArgs
 
   @Args4JOption(
     name = "--loci-partitioning",
-    usage = "If set: load a LociPartitioning from this path if it exists; write a computed partitioning to this path if it doesn't.",
+    usage = "Load a LociPartitioning from this path if it exists; else write a computed partitioning to this path.",
     handler = classOf[StringOptionHandler]
   )
-  var lociPartitioningPathOpt: Option[String] = None
+  protected var _lociPartitioningPathOpt: Option[String] = None
+
+  def lociPartitioningPathOpt: Option[String] = _lociPartitioningPathOpt
```

Comment:
makes sense, added! the getter interface allows for different path-resolution logic in subclasses, e.g. PartitionedRegionsArgs